pytomorrowio/fields.py

```diff
@@ -0,0 +1,232 @@
+"""Tomorrow.io field definitions"""
```

Comment:
Moved these to separate file as they are an implementation detail, not a value that clients should need to use.